contents/docs/experiments/experiment-significance.mdx

```diff
@@ -0,0 +1,103 @@
+
+#### 1. Probability of being the best
+
+We use Monte Carlo simulations to determine the probability of each variant being the best. Each variant can be modeled as a gamma distribution, with the shape parameter equal to the trend count and the exposure parameter equal to the relative exposure for that variant. For each variant, we sample from their respective distributions to get a count value. We perform 100,000 simulation runs in our calculations.
+
+The probability of a variant being the best is given by:
+
+<FormulaScreenshot
+  imageLight={TrendExperimentCalculationLight}
+  imageDark={TrendExperimentCalculationDark}
+  alt="Trend experiment calculation" 
+  classes="rounded"
+/>
+
+>**Trend experiment exposure**
+>
+>
+>Trend experiments compare counts of events. Since count data can refer to the total count of events or the number of unique users, we use a proxy metric to measure exposure. The number of times the `feature_flag_called` event returns control or test is used as the respective exposure for the variant. This event is sent automatically when you call `posthog.getFeatureFlag()`.
+>
+>It's important to note that a variant showing fewer count data can still have a higher probability of being the best if its exposure is much smaller. This is because the relative exposure is taken into account when calculating probabilities.
+
+#### 2. Statistical significance
+
+To calculate significance, we measure p-values using a [Poisson means test](https://www.evanmiller.org/statistical-formulas-for-programmers.html#count_test). Results are significant when the p-value is less than 0.05
+
+## How do we determine final significance?
+
+For your results and conclusions to be valid, any experiment must have significant exposure. For instance, if you test a product change and only one user sees the change, you can't extrapolate from that single user that the change will be beneficial or detrimental for your entire user base. This principle holds true for any simple randomized controlled experiment, such as those used in testing new drugs or vaccines.
+
+Even with a large sample size (e.g., approximately 10,000 participants), results can still be ambiguous. For example, if the difference in conversion rates between variants is less than 1%, it becomes difficult to determine if one variant is truly better than the other. To achieve statistical significance, there must be a sufficient difference between the conversion rates given the exposure size.
+
+PostHog computes this statistical significance for you automatically. We notify you when your experiment has reached statistically significant results, making it safe to draw conclusions and terminate the experiment.
+
+In the early days of an experiment, data can vary wildly, and sometimes one variant can seem overwhelmingly better. In this case, our significance calculations might say that the results are significant, but this shouldn't be the case, since we need more data.
```

Comment:
```suggestion
In the early days of an experiment, data can vary wildly and sometimes one variant can seem overwhelmingly better. In this case, our significance calculations might say that the results are significant, but this shouldn't be the case, since we need more data.
```